cmd/libs/go2idl/import-restrictions

```diff
@@ -0,0 +1,11 @@
+{
+  "Rules": [
+    {
+      "SelectorRegexp": "k8s[.]io",
```

Comment:
".import-restrictions"? That's fine, I was thinking it should be more visible, but I don't feel too strongly.